src/components/Header/Header.js

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { Component } from 'react';
 import HeaderNav from './HeaderNav/HeaderNav';
+import HeaderUserMenu from './HeaderUserMenu/HeaderUserMenu';
+import { ReactComponent as DopplerLogo } from '../../doppler-logo.svg';
+
+class Header extends Component {
```

Comment:
If you are already importing global object (_React_) why not using this in place importing another name:

```suggestion
// import React from 'react';
class Header extends React.Component {
```

Another option is not importing global React, for example:


```
import { Component, AnotherThing, AnotherMore } from 'react';
```

Reply:
i didn't know that. thanks! good catch! Which you prefeer?

Reply:
Mmm, I think that _Nicholas Zakas_ prefers the last one, let me research...

Reply:
Well he talks about exports, but, I think that the same applies for imports: https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/

So, what do you think about **avoiding import defaults when it is possible**?

Reply:
Why not make this component as a function? For les overhead `class Header extends React.Component {`